include/deal.II/particles/particle_handler.h

```diff
@@ -334,7 +334,8 @@ namespace Particles
      * @authors Luca Heltai, Bruno Blais, 2019.
      */
     template <class VectorType>
-    void
+    typename boost::disable_if<
```

Comment:
Why not just
```suggestion
    typename std::enable_if<!
```
?

Reply:
I thought `disable_if` was more self documenting (I often miss the `!`). Changed.